spec/services/shared_examples_preserved_object_handler.rb

```diff
@@ -71,17 +71,20 @@
 
 RSpec.shared_examples 'druid not in catalog' do |method_sym|
   let(:druid) { 'rr111rr1111' }
-  let(:exp_msg) { "#{exp_msg_prefix} #<ActiveRecord::RecordNotFound: Couldn't find PreservedObject> db object does not exist" }
+  # let(:exp_msg) { "#{exp_msg_prefix} #<ActiveRecord::RecordNotFound: Couldn't find PreservedObject> db object does not exist" }
```

Comment:
whoops, looks like we left in a bit of commented code for an expectation that we loosened slightly (since sometimes this error message comes as a result of a trapped exception, and sometimes as a result of a pro-active check).  happy to either fix this PR or remove in a tiny follow-on PR.  ditto the next two lines of commented code in this file.

Reply:
Slight preference for you to fix it here.